pkgs/top-level/all-packages.nix

```diff
@@ -6467,6 +6467,17 @@ with pkgs;
 
   libskk = callPackage ../development/libraries/libskk { };
 
+  mozc-ut = mozc.override { dictionaries = [
```

Comment:
If `fcitx-mozc-ut` is a separate file in `pkgs/by-name`, I think `mozc-ut` should be a separate file too.

Reply:
Done!